tutorials/examples/cov2d_experiment.py.dontrun

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env  python3.6
+"""
+This script illustrates denoising 2D images using batched Cov2D class
+from experime dataset and outputing to mrcs file.
+"""
+
+from aspire.basis.ffb_2d import FFBBasis2D
+from aspire.denoising.denoiser_cov2d import DenoiserCov2D
+from aspire.estimation.noise import AnisotropicNoiseEstimator
+from aspire.source.relion import RelionSource
+
+
+# Set input path and files and initialize other parameters
+data_folder = '/path/to/untarred/empiar/dataset/'
```

Comment:
Can we make these parameters capitalized?

Reply:
Done in commit 0e187b3.